bindings/rascal/models/gaptools.py

```diff
@@ -244,7 +244,7 @@ def fit_gap_simple(
     energies,
     kernel_energies_sparse,
     energy_regularizer_peratom,
-    energy_atom_contributions,
+    energy_atom_contributions=None,
```

Comment:
I'd actually rather explicitly set it to zero, because that makes it clear that there's always a baseline (it's just that it can often be zero).

Reply:
That would also avoid a lot of the extra logic implemented further down here.